content/en/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -197,23 +197,72 @@ As Pod specs with GMSA fields populated (as described above) are applied in a cl
 
 1. The container runtime configures each Windows container with the specified GMSA credential spec so that the container can assume the identity of the GMSA in Active Directory and access services in the domain using that identity.
 
+## Containerd 
+
+On Windows Server 2019, in order to use GMSA with Containerd, you must be running OS Build 17763.1817 (or later) which can be installed using the patch [KB5000822](https://support.microsoft.com/en-us/topic/march-9-2021-kb5000822-os-build-17763-1817-2eb6197f-e3b1-4f42-ab51-84345e063564). 
+
+There is also a known issue with containerd when trying to connect to SMB shares from Pods. Once you have configured GMSA, the pod will be unable to connect to the share using the Hostname or FQDN but connecting via IP address works as expected. 
+
+```PowerShell
+ping adserver.ad.local
+```
+Resolves the Hostname to IP Address Correctly and will produce an output similar to this:
+```
+```PowerShell
+Pinging adserver.ad.local [192.168.111.18] with 32 bytes of data:
+Reply from 192.168.111.18: bytes=32 time=6ms TTL=124
+Reply from 192.168.111.18: bytes=32 time=5ms TTL=124
+Reply from 192.168.111.18: bytes=32 time=5ms TTL=124
+Reply from 192.168.111.18: bytes=32 time=5ms TTL=124
+```
+
+However, when attempting to browse the directory using the hostname
+
+```PowerShell
+cd \\adserver.ad.local\test
+```
+
+It will appear that the share doesn't exists:
+
+```PowerShell
+cd : Cannot find path '\\adserver.ad.local\test' because it does not exist.
+At line:1 char:1
++ cd \\adserver.ad.local\test
++ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    + CategoryInfo          : ObjectNotFound: (\\adserver.ad.local\test:String) [Set-Location], ItemNotFoundException
+    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.SetLocationCommand
+```
+
+While trying to browse to the share using the IP Address instead:
+
+```PowerShell
+cd \\192.168.111.18\test
+```
+
+Changes to the directory as expected and you should see a prompt similar to this:
+```PowerShell
+Microsoft.PowerShell.Core\FileSystem::\\192.168.111.18\test>
+```
+
+To correct the behaviour you must run the following `reg add "HKLM\SYSTEM\CurrentControlSet\Services\hns\State" /v EnableCompartmentNamespace /t REG_DWORD /d 1` and restart the pods to add the required registry key.
```

Comment:
Can you clarify that this needs to be run on the node?

Reply:
What does “restart the pods” mean? Is it in fact Pod deletion followed by the control plane autohealing your workload?